### PR TITLE
Fixes #3237 - add --reset-db-feature

### PR DESCRIPTION
**Pull request by ares**
Depends on https://github.com/theforeman/puppet-foreman/pull/116, https://github.com/theforeman/kafo/pull/31 and new kafo release

EDIT: ready to be merged, please review

### CODE REVIEW

**Comment by ares** [comment]
this will have to be rebased after #75

**Comment by ares** [comment]
rebased, please review

File: bin/foreman-installer [hunk 1]
```diff
@@ -4,9 +4,41 @@ require 'highline/import'
...
+  return false if mod.nil?
+  mod.enabled?
+end
+
+def get_param(mod, name)
+  @result.param(mod, name).value
+end
+
+# functions specific to foreman installer
+Kafo::KafoConfigure.app_option '--reset-foreman-db', :flag, 'Drop foreman database first? You will loose all data!', :default => false
```
Comment (domcleal) [inline bin/foreman-installer]: s/loose/lose/

**Comment by domcleal** [comment]
Untested, but reads OK :+1:

**Comment by domcleal** [comment]
Could you rebase please?

**Comment by ares** [comment]
Rebased

**Comment by domcleal** [comment]
115208 < gwmngilfen> the comment change at the top of 71 will affect the debian patches, so i'd like to see a scratchbuild of that

**Comment by GregSutcliffe** [comment]
:+1: works on Debian (apart from http://theforeman.org/issues/3610)

**Comment by domcleal** [comment]
Merged as 56b726d, thanks @ares!

**Pull request closed without merging**